test/studies/IMSuite/vertexColor/vertexColoring-serial.chpl

```diff
@@ -22,7 +22,7 @@
 pragma "error mode fatal"
 module vertexColoring {
 
-  use Random, IO;
+  use Random, IO, Math;
```

Comment:
This PR didn't cause this, but since you are editing it anyways can you fix the spacing here?